docs: consolidate getting started and context (#44)
podium-lib/podium-lib.github.io: docs/podlet/proxying.md

title: Proxying
---

> Proxying is useful in cases where you decide to only publicly expose layout servers to incoming traffic.

Reviewer: ```suggestion
Proxying is useful in cases where you decide to only expose layout servers to publicly incoming traffic.
```

Author: How about this instead?

```suggestion
Proxying is useful in cases where you decide to only expose layout servers directly to the Internet.
```

Reviewer: deal 👍

app.listen(7100);
```

> ## Public podlets and CORS
> 
> If your infrastructure is set up so podlet servers are publicly available you can choose to communicate with podlet servers directly by enabling [cross-origin resource sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). You can still use the Podium proxy if you prefer.

Reviewer: ```suggestion
## Public podlets and Cross-Origin Resource Sharing (CORS)

If your infrastructure is set up so podlet servers are publicly available you can choose to communicate with podlet servers directly by enabling [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). You can still use the Podium proxy if you prefer.
```
